README.md

```diff
@@ -1,4 +1,7 @@
-# codebuild-webdeploy-package
+<!-- BEGIN_TF_DOCS -->
+# terraform-aws-iam-role-vmimport
+GitHub: [StratusGrid/terraform-aws-iam-role-vmimport](https://github.com/StratusGrid/terraform-aws-iam-role-vmimport)
+
```

Comment:
Seems like a copy/paste mistake here. This repo is not for vmimport.

Reply:
Fixed